akka-actor/src/main/scala/akka/actor/CoordinatedShutdown.scala

```diff
@@ -140,6 +140,27 @@ object CoordinatedShutdown extends ExtensionId[CoordinatedShutdown] with Extensi
    */
   def clusterDowningReason: Reason = ClusterDowningReason
 
+  /**
+    * Scala API: The shutdown was initiated by a failure to join a seed node.
+    */
+  object JoiningSeedNodesUnsuccessfulReason extends Reason
```

Comment:
That seems extremely specific, should it rather be `ClusterJoinUnsuccessful` or something like that, or is it important that it was specifically joining to seed nodes that failed?

Reply:
I agree, could be slightly less specific

Reply:
SGTM